compiler/rustc_target/src/spec/mipsel_unknown_none.rs

```diff
@@ -0,0 +1,32 @@
+// Bare MIPS32r2, little endian, softfloat
+// can be used for MIPS M4K core (e.g. on PIC32MX devices)
+
+use crate::spec::{LinkerFlavor, LldFlavor, RelocModel};
+use crate::spec::{PanicStrategy, Target, TargetOptions};
+
+pub fn target() -> Target {
+    Target {
+        llvm_target: "mipsel-unknown-none".to_string(),
+        target_endian: "little".to_string(),
+        pointer_width: 32,
+        target_c_int_width: "32".to_string(),
+        data_layout: "e-m:m-p:32:32-i8:8:32-i16:16:32-i64:64-n32-S64".to_string(),
```

Comment:
Did you build this yourself or is it what clang uses?

Reply:
This was "stolen" from the mipsel_unknown_linux targets